### PR TITLE
feat: add a new api to get rag evaluation report

### PR DESCRIPTION
**Pull request by 0xff-dev**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it
feat: add a new api to get rag evaluation report

#### Which issue(s) this PR fixes

Fix https://github.com/kubeagi/arcadia/issues/684

#### Special notes for your reviewer
image

### CODE REVIEW

File: apiserver/pkg/rag/report.go [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+	green    = "green" // 好
+
+	summarySuggestionTemplate = `通过此次评估，您的智能体得分偏低，主要体现在 <strong>%s</strong> 这 %d 项指标得分偏低。
+<br>
+<strong>建议您对特定场景应用的模型进行模型精调；%s。</strong>`
+	noSuggestionTempalte = `通过此次评估，您的 RAG 方案得分 <span style="color:orange">%.2f</span>`
+)
+
+var (
+	metricChinese = map[string]string{
```
Comment (bjwswang) [inline apiserver/pkg/rag/report.go:55]: also metricEnglish

File: apiserver/pkg/rag/report.go [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+	blueColorEnv = "BLUE_ENV"
+	blue         = "blue" // 散点图的颜色
+
+	orangeEnv = "ORANGE_RNV"
+	orange    = "orange" // 差
+
+	greenEnv = "GREEN_ENV"
+	green    = "green" // 好
+
+	summarySuggestionTemplate = `通过此次评估，您的智能体得分偏低，主要体现在 <strong>%s</strong> 这 %d 项指标得分偏低。
```
Comment (bjwswang) [inline apiserver/pkg/rag/report.go:48]: summarySuggestionTemplateInEng

File: apiserver/pkg/rag/report.go [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+	ReportDetail struct {
+		Data  []ReportLine `json:"data"`
+		Total int          `json:"total"`
+	}
+)
+
+func ParseSummary(
+	ctx context.Context, c client.Client,
+	appName, ragName, namespace string,
+	metricThresholds map[string]float64) (Report, error) {
```
Comment (bjwswang) [inline apiserver/pkg/rag/report.go:125]: A new parameter `language`

**Review by bjwswang — CHANGES_REQUESTED** [review]
@0xff-dev we only provide the summary data. Let the front-end decides how to use them.

**Comment by 0xff-dev** [comment]
For the English message, I think it can be implemented in the next version

**Pull request merged**